FitTheWall/Assets/PositionTo.cs

```diff
@@ -0,0 +1,12 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PositionTo : MonoBehaviour
+{
+    public GameObject objectToTrack;
```

Comment:
comment variable.

Also, does this really need to be public? Let me know.

I bet we can make it a "[SerializeField] private"

Reply:
@Spraxs It's public in order to be able to drag in objects from inside the Unity editor.

Reply:
Try [SerializeField] private